### ISSUE TITLE
Set up automatic backup on the server

### ISSUE THREAD

**Issue by rossant** [cortex-lab/alyx#127]
Set up a cron job to run `alyx/backups/backup-tsv.sh /path/to/backup_dir/` every day or whatever. This script creates a `alyx-backups/yyyy-mm-dd/` subfolder with TSV backups, and it also updates the `Alyx Backup` Google sheet.

**Comment by nsteinme** [comment]
I tried to do this, see /etc/crontab on rod.cortexlab. However it doesn't
seem to have been working.

**Comment by rossant** [comment]
Yes it's not obvious to set up correctly. Maybe check the chmod. Also I think there is a command somewhere to check the cron manually...

**Comment by nippoo** [comment]
Hmmm we really shouldn't be running this as root. Should probably be in www-data's crontab. Will look into it

**Comment by nsteinme** [comment]
Hm well currently backup.sh isn't even running, I get this error, Max I thought you made alyx_ro working again? I can check it when I get into lab tomorrow, can't use it from home anyway. Another problem seems to be that zserver maybe isn't mounted properly on rod - /mnt/zserver/Data isn't something I can cd into. 

```
nick@rod:/data/www/alyx$ ./backups/backup-tsv.sh /mnt/zserver/Data/
mkdir: cannot create directory ‘/mnt/zserver/Data’: File exists
cat: queries/*.sql: No such file or directory
psql: FATAL:  no pg_hba.conf entry for host "128.40.89.151", user "alyx_ro", database "alyx", SSL on
FATAL:  no pg_hba.conf entry for host "128.40.89.151", user "alyx_ro", database "alyx", SSL off
Backup done in /mnt/zserver/Data//alyx-backups/2017-03-08/
./backups/backup-tsv.sh: line 21: ./upload-gsheets.py: No such file or directory
```

**Comment by nsteinme** [comment]
Could you open a new issue about the documentation so we don't forget? In
the doc, mention both where the backups go and also what is running them?
